runtime/drivers/duckdb/transporter_mysql_to_duckDB_test.go

```diff
@@ -123,3 +132,55 @@ func allMySQLDataTypesTest(t *testing.T, db *sql.DB, dsn string) {
 	require.NoError(t, res.Close())
 	require.NoError(t, to.Close())
 }
+
+func mysqlToDuckDB(t *testing.T, dsn string) {
```

Comment:
Can we move this test to a test file that starts with `model_executor_` instead? For when we drop the `transporter` files?

Reply:
Actually I planned to rename this file and drop the transporter test when re drop the transporter files. Till then kept it in the same file to avoid creating more test containers and keep test time short.
I am okay to also create new test if you prefer that way.